nautobot/dcim/migrations/0030_migrate_region_and_site_data_to_locations.py

```diff
@@ -48,14 +54,17 @@ def create_region_location_type_locations(region_class, location_class, region_l
     )
     for region in regions:
         try:
-            location_class.objects.create(
+            region_loc = location_class.objects.create(
+                id=region.id,
                 location_type=region_lt,
                 name=region.name,
                 description=region.description,
                 parent=location_class.objects.get(name=region.parent.name, location_type=region_lt)
```

Comment:
Since the UUID is being preserved?

```suggestion
                parent=location_class.objects.get(id=region.parent_id)
```